crates/bevy_asset/src/loader.rs

```diff
@@ -168,7 +168,7 @@ impl<T: Resource> Default for AssetLifecycleChannel<T> {
     }
 }
 
-/// Reads [AssetResult]s from an [AssetChannel] and updates the [Assets] collection and [LoadState] accordingly
+/// Reads [AssetResult]s from an [AssetLifecycleChannel] and updates the [Assets] collection and [crate::LoadState] accordingly
```

Comment:
This is still true, but it is now a subset of what is done. I think this should probably say something a bit more generic like:

> Updates the [Assets] collection according to the changes queued up by [AssetServer]

Reply:
Added this change.